spec/controllers/steps/case/charges_controller_spec.rb

```diff
@@ -4,6 +4,63 @@
   let(:form_class) { Steps::Case::ChargesForm }
   let(:decision_tree_class) { Decisions::CaseDecisionTree }
 
+  let(:crime_application) { CrimeApplication.create }
```

Comment:
If possible let's wrap these contexts and their associated lets in a new describe block as the lets are not needed for the other scenarios (`#edit` and `#update`) and make tests more isolated and less prone to clashing.